console/console-init/ui/src/components/DividerWithHeading/DividerWithHeading.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Divider } from "@patternfly/react-core";
+import { StyleSheet } from "@patternfly/react-styles";
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
can we rename this ?

Reply:
don't get you. are you asking for container rename or something else ?